### ISSUE TITLE
How to intstall?

### ISSUE THREAD

**Issue by dirkjlouw** [NSSpiral/Blackb0x#54]
Hope somebody can assist me, I am trying very hard to follow the instructions on to install.

Running iMac 2020 with this version installed: 
https://github.com/NSSpiral/Blackb0x/releases/download/0.5/Blackb0x.app.zip

ATV 3 A1427
I start up the ATV until it is booted and I can see apple tv icons.
(I cannot connect HDMI and USB as the adapters are too thick.)
Unplug HDMI and plug in micro USB.
Then right click and open Blackb0x. 
I can then click Install Jailbreak.
follow the prompts. to get into DFU.
First DOWN and MENU.
Then MENU and PLAY/PAUSE.
Click on the apple TV icon for the blackb0x application again.
Then click on install Jailbreak. 
Wait a few seconds and click on the appleTv again.
Then get a Jailbreak installed. 

what do I do from here? if I unplug the USB and plug in the hdmi, nothing. if I switch off the ATV and boot again its back to appleTv.

I am stuck here.
<img width="812" alt="Screenshot 2021-08-22 at 15 10 04" src="https://user-images.githubusercontent.com/11453406/130356472-30f23c00-65fc-454a-a756-a0eb44e5e65d.png">

**Comment by dannyboyswe** [comment]
Joshtv.net is down (from which Kodi and NitoTV is downloaded) and has been down for some time, so there is no way to install Kodi or NitoTV right now. I believe they are working on a fix.

**Comment by XVincentX** [comment]
Did this get fixed? I believe I am experiencing the same issue

**Comment by Vipizz** [comment]
A bit confused... Do you mean dirkjlouw's issue or that joshtv.net is down?

dirkjlouw's issue probably that he missed step 0 in manual.

Joshtv.net down issue... look at info in issue #51.

**Comment by XVincentX** [comment]
I was referring to the fact that JoshTV is down. I’ll take a look at the additional steps and see if that solves the issue. Thanks!

V.

**Comment by dirkjlouw** [comment]
> A bit confused... Do you mean dirkjlouw's issue or that joshtv.net is down?
> 
> dirkjlouw's issue probably that he missed step 0 in manual.
> 
> Joshtv.net down issue... look at info in issue #51.

How would you say I missed step 0? What is step 0 that you are referring too?

**Comment by Vipizz** [comment]
You got ATV3.1 (A1427)

Step 0 must be done before jailbreak =
(3,1 Only) PWN with Arduino + synackuk's fork of checkm8-A5

**Comment by dirkjlouw** [comment]
> You got ATV3.1 (A1427)
> 
> Step 0 must be done before jailbreak =
> (3,1 Only) PWN with Arduino + synackuk's fork of checkm8-A5

Would you mind to explain that step please? 
I missed that step completly, I also have no clue. Would that be a app I need to install seperate or do I download arduino application for mac and get the code from https://github.com/synackuk/checkm8-a5

If I do get this step correct, joshTv being down, would I still be able to install kodi?

**Comment by Whosdrunk** [comment]
I have the same issue with A1427. As soon as jailbreak is installed, my device says Discovery Mode. The screen says, connect usb, nothing happens, even of I reconnect. When I I restart the box with usb hdmi plugged, ir boots into the normal screen. Eben After hours nothing hapoens :( Andy ideas?

**Comment by Kabre555** [comment]
Once the jailbreak is installed you can SSH into the apple tv and manually install Kodi (search for the commands), the automated install is not working.

**Comment by Whosdrunk** [comment]
Hey, thx :-) I didn´t read/find that, perfect. So, this I can also do via an windows machine, ok?
As am not good in this : Is there an instruction how I find and do that via SSH? Thank you so much,

**Comment by pulkitraggarwal** [comment]
> Once the jailbreak is installed you can SSH into the apple tv and manually install Kodi (search for the commands), the automated install is not working.

I try to ssh into the apple tv but it says connection refused. Any suggestions how to proceed?

**Comment by Kabre555** [comment]
I think you have to enable home sharing first

**Comment by Whosdrunk** [comment]
I try to connect via ssh (putty or via cmd), connection refused. I activated home sharing, but it will not work. Any ideas?

**Comment by mcr-ksh** [comment]
+1
